hyperspy/tests/io/test_edax.py

```diff
@@ -0,0 +1,217 @@
+import nose.tools as nt
+
+import gzip
+import hashlib
+import os.path
+import os
+import shutil
+import numpy as np
+
+from hyperspy.io import load
+from hyperspy import signals
+
+test_files = ['Live Map 2_Img.ipr',
+              'single_spect.spc',
+              'spd_map.spc',
+              'spd_map.spd']
+my_path = os.path.dirname(__file__)
+
+
+class TestSpcSpectrum:
+
+    def setUp(self):
+        print('testing single spc spectrum...')
+        self.spc = load(os.path.join(
+            my_path,
+            "edax_files",
+            test_files[1]))
+
+    def test_data(self):
+        nt.assert_equal(np.uint32, self.spc.data.dtype)     # test datatype
+        nt.assert_equal((4096,), self.spc.data.shape)       # test data shape
+        nt.assert_equal(
+            [0, 0, 0, 0, 0, 0, 1, 2, 3, 3, 10, 4, 10, 10, 45, 87, 146, 236,
+             312, 342], self.spc.data.tolist()[:20])   # test 1st 20 datapoints
+
+    def test_parameters(self):
+        elements = self.spc.metadata.as_dictionary()['Sample']['elements']
+        sem_dict = self.spc.metadata.as_dictionary()[
+            'Acquisition_instrument']['SEM']
+        eds_dict = sem_dict['Detector']['EDS']
+        signal_dict = self.spc.metadata.as_dictionary()['Signal']
+
+        # Testing SEM parameters
+        nt.assert_almost_equal(22, sem_dict['beam_energy'])
+        nt.assert_almost_equal(0, sem_dict['tilt_stage'])
+
+        # Testing EDS parameters
+        nt.assert_almost_equal(0, eds_dict['azimuth_angle'])
+        nt.assert_almost_equal(34, eds_dict['elevation_angle'])
+        nt.assert_almost_equal(129.31299, eds_dict['energy_resolution_MnKa'],
+                               places=5)
+        nt.assert_almost_equal(50.000004, eds_dict['live_time'], places=6)
+
+        # Testing elements
+        nt.assert_equal(['Al', 'C', 'Ce', 'Cu', 'F', 'Ho', 'Mg', 'O'],
+                        sorted(elements))
```

Comment:
Aren't elements already sorted?

Depending on what you want to do specifically, I would:
 - Sort your manually typed list (just in case the sorting changes for some reason or someone swapped the order in the list by accident when adding something in the future)
 - If not important that the elements are in particular order, test sets instead of lists: `nt.assert_set_equal({'Al', 'C', ...}, set(elements))`